docs/v2/4_erc20-interactions.md

```diff
@@ -107,7 +113,7 @@ public class ERC20NameExample : MonoBehaviour
 
 ### Symbol {#symbol}
 
-Returns the symbol of the token, e.g. "USDC".
+Returns the symbol of an ERC-20 token associated with a specified contract address (e.g. "USDC").
```

Comment:
For example, shouldn't this then also be USD Coin instead of USDC?

Reply:
Cleared with @waymobetta. name and symbol are separate params
![image](https://user-images.githubusercontent.com/36713098/221240792-5f94b2ad-b219-4ecf-905e-2db84d60c79b.png)